README.md

```diff
@@ -175,18 +175,28 @@ number and the deployment network configuration allows connections on the
 configured port. As noted above, the final caveat is that in non-Kube
 deployments, port 9091 is expected for automatic port-scanning target discovery.
 
+## EVENT TEMPLATES
+
+JDK Flight Recorder has a concept of "event templates", which are a sort of
```

Comment:
I'd prefer to be more concise, e.g.

JDK Flight Recorder event templates (.jfc files) are...

So things like "has a concept of..." or "are a sort of..." should be removed.